data/json/furniture.json

```diff
@@ -2408,7 +2423,7 @@
     "type": "furniture",
     "id": "f_mannequin",
     "name": "mannequin",
-    "description": "Put clothes on it and wish you looked as good.",
+    "description": "Put clothes on it, talk to it.  Who's around to judge you?  Wait...did it just move?",
```

Comment:
[Spacing for ellipses is pure style](https://www.grammarbook.com/punctuation/ellipses.asp) but I'd go with whatever the majority has been used.

Reply:
Totally on board with following style...I just don't know what it is?

Reply:
`wait... Did it just move?` would be the standard